elastic/kibana
# [Infra] [APM] Remove `new` badges for profiling in APM and infra

Closes #175671 
## Summary

This PR removes all `new` badges for the profiling feature inside Infra and APM UI
Example:
| Before | After |
| ------ | ------ |
| ![image](https://github.com/elastic/kibana/assets/14139027/c9a78a6d-09d5-49bf-9dde-b79b5cf7a4af) | ![image](https://github.com/elastic/kibana/assets/14139027/faea5963-100b-4897-a4a6-48855002ff4c) |

## Testing
- Open Hosts View > host flyout 
   - No `new` badges should be present for the Profiling tab/link
    ![image](https://github.com/elastic/kibana/assets/14139027/42f5c74e-e321-41c8-bd64-75b778001912)

   - Click on `open as page` - the same should apply
    ![image](https://github.com/elastic/kibana/assets/14139027/7a9448c1-6165-46de-9468-cf406695cac4)

 - Check the same in Inventory  > host flyout 
      <img width="1920" alt="image" src="https://github.com/elastic/kibana/assets/14139027/a7085c42-f2fb-4fca-96f8-df754ea22572">

- Open a service in APM > Service Overview
    - No `new` badges should be present for the Profiling tab
     ![image](https://github.com/elastic/kibana/assets/14139027/e0863d25-3544-4de8-a9e3-d1d175c70eff)

    - Same for an APM transaction
    ![image](https://github.com/elastic/kibana/assets/14139027/a7f06ac8-128a-42c3-9506-4df523883379)

    - Open an APM transaction and scroll down and click on investigate  
       - No `new` badges should be present for the Profiling links
       ![image](https://github.com/elastic/kibana/assets/14139027/aafa18f7-46d7-4b18-bec1-7acc9ed8a89e)
